sympy/core/tests/test_expr.py

```diff
@@ -1652,15 +1652,15 @@ def test_random():
 def test_round():
     from sympy.abc import x
 
-    assert Float('0.1249999').round(2) == 0.12
+    assert str(Float('0.1249999').round(2)) == '0.12'
```

Comment:
Can we use comp rather than str here?

Reply:
(and below)

Reply:
when testing round I want to make sure that exactly these digits are shown